sentry_sdk/client.py

```diff
@@ -248,6 +248,19 @@ def _prepare_event(
                     )
             event = new_event  # type: ignore
 
+        before_send_transaction = self.options["before_send_transaction"]
+        if before_send_transaction is not None and event.get("type") == "transaction":
+            new_event = None
+            with capture_internal_exceptions():
+                new_event = before_send_transaction(event, hint or {})
+            if new_event is None:
+                logger.info("before send transaction dropped event (%s)", event)
+                if self.transport:
+                    self.transport.record_lost_event(
+                        "before_send_transaction", data_category="error"
```

Comment:
```suggestion
                        "before_send", data_category="transaction"
```

data_category is self evident
for reason, see [develop docs](https://develop.sentry.dev/sdk/client-reports/#envelope-item-payload) that say that the valid reasons list is in [snuba](https://github.com/getsentry/snuba/blob/4e7cfdddcf7b93eacb762bc74ca2461cec9464e5/snuba/datasets/outcomes_processor.py#L24-L34).